src/supertux/game_object_manager.cpp

```diff
@@ -69,6 +69,30 @@ GameObjectManager::process_resolve_requests()
   m_name_resolve_requests.clear();
 }
 
+void
+GameObjectManager::try_process_resolve_requests()
+{
+  assert(m_gameobjects_new.empty());
+  std::vector<GameObjectManager::NameResolveRequest> new_list;
+
+  for (const auto& request : m_name_resolve_requests)
+  {
+    GameObject* object = get_object_by_name<GameObject>(request.name);
```

Comment:
Can you maybe change this to 
```c++
auto object = ...
```

Reply:
`auto*` ?

Our guidelines recomment receiving pointers as `auto*`

Reply:
Interesting. https://stackoverflow.com/questions/12773257/does-auto-type-assignments-of-a-pointer-in-c11-require

I believe our guidelines should be changed, but I will have to discuss that with the rest of the programmers.